src/System.Security.Cryptography.Pkcs/ref/System.Security.Cryptography.Pkcs.cs

```diff
@@ -48,6 +48,7 @@ public sealed partial class AlgorithmIdentifier
         public AlgorithmIdentifier(System.Security.Cryptography.Oid oid, int keyLength) { }
         public int KeyLength { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public System.Security.Cryptography.Oid Oid { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
+        public byte[] Parameters { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Comment:
This CompilerGeneratedAttribute isn't necessary here or on the rest of these.  @bartonjs / @weshaggard, are these being generated by the ref creation tool?

Reply:
I do believe that is where the existing ones came from, yes.

Reply:
We can re-run the generation for this ref to normalize it. Just run this target https://github.com/dotnet/corefx/blob/master/dir.targets#L123 on the ref project and these attributes should be excluded (https://github.com/dotnet/corefx/blob/master/tools-local/DefaultGenApiDocIds.txt#L20).